src/locales/en/translation.json

```diff
@@ -248,7 +248,12 @@
   },
   "footer": {
     "github": "Oasis Wallet is fully <0>open source</0> - Feedback and issues are appreciated!",
+    "poweredBy": "Powered by {{token}}",
```

Comment:
@tjanez  Do we want `"Powered by {{backendName}} API & SimplyVC's gRPC"`?

Reply:
Had a meeting with Tadej, yep that's his preference. Thx